emscripten.py

```diff
@@ -1108,6 +1110,8 @@ def make_emulated_param(i):
                                 'shiftRightLogicalByScalar',
                                 'shiftLeftByScalar'];
     fundamentals = ['Math', 'Int8Array', 'Int16Array', 'Int32Array', 'Uint8Array', 'Uint16Array', 'Uint32Array', 'Float32Array', 'Float64Array', 'NaN', 'Infinity']
+    if settings['USE_PTHREADS']:
+      fundamentals += ['Atomics', 'SharedInt8Array', 'SharedInt16Array', 'SharedInt32Array', 'SharedUint8Array', 'SharedUint16Array', 'SharedUint32Array', 'SharedFloat32Array', 'SharedFloat64Array']
```

Comment:
the non-shared are not really needed, I think? if we can remove them, it would avoid wasting minified names in the asm scope (and since they are first, they get the good names like `a,b,c`...)